x-pack/plugin/esql/qa/testFixtures/src/main/resources/bucket.csv-spec

```diff
@@ -0,0 +1,460 @@
+4              |1985-11-01T00:00:00.000Z
+//end::docsBucketMonthlyHistogram-result[]
+;
+
+docsBucketWeeklyHistogram#[skip:-8.13.99, reason:BUCKET renamed in 8.14]
+//tag::docsBucketWeeklyHistogram[]
+FROM employees
+| WHERE hire_date >= "1985-01-01T00:00:00Z" AND hire_date < "1986-01-01T00:00:00Z"
+| EVAL week = BUCKET(hire_date, 100, "1985-01-01T00:00:00Z", "1986-01-01T00:00:00Z")
+| STATS hires_per_week = COUNT(*) BY week
+| SORT week
+//end::docsBucketWeeklyHistogram[]
+;
+
+//tag::docsBucketWeeklyHistogram-result[]
+  hires_per_week:long    |    week:date
+2              |1985-02-18T00:00:00.000Z
+1              |1985-05-13T00:00:00.000Z
+1              |1985-07-08T00:00:00.000Z
+1              |1985-09-16T00:00:00.000Z
+2              |1985-10-14T00:00:00.000Z
+4              |1985-11-18T00:00:00.000Z
+//end::docsBucketWeeklyHistogram-result[]
+;
+
+docsBucketLast24hr#[skip:-8.13.99, reason:BUCKET renamed in 8.14]
+//tag::docsBucketLast24hr[]
+FROM sample_data 
+| WHERE @timestamp >= NOW() - 1 day and @timestamp < NOW()
+| EVAL bucket = BUCKET(@timestamp, 25, NOW() - 1 day, NOW())
+| STATS COUNT(*) BY bucket
+//end::docsBucketLast24hr[]
+;
+
+   COUNT(*):long    |    bucket:date
+;
+
+docsGettingStartedBucket#[skip:-8.13.99, reason:BUCKET renamed in 8.14]
+// tag::gs-bucket[]
+FROM sample_data
+| KEEP @timestamp
+| EVAL bucket = BUCKET(@timestamp, 24, "2023-10-23T00:00:00Z", NOW())
+// end::gs-bucket[]
+| LIMIT 0
+;
+
+@timestamp:date | bucket:date
+;
+
+docsGettingStartedBucketStatsBy#[skip:-8.13.99, reason:BUCKET renamed in 8.14]
+// tag::gs-bucket-stats-by[]
+FROM sample_data
+| KEEP @timestamp, event_duration
+| EVAL bucket = BUCKET(@timestamp, 24, "2023-10-23T00:00:00Z", "2023-10-23T23:59:59Z")
+| STATS COUNT(*) BY bucket
+// end::gs-bucket-stats-by[]
+| SORT bucket
+;
+
+COUNT(*):long | bucket:date
+2              |2023-10-23T12:00:00.000Z
+5              |2023-10-23T13:00:00.000Z
+;
+
+docsGettingStartedBucketStatsByMedian#[skip:-8.13.99, reason:BUCKET renamed in 8.14]
+// tag::gs-bucket-stats-by-median[]
+FROM sample_data
+| KEEP @timestamp, event_duration
+| EVAL bucket = BUCKET(@timestamp, 24, "2023-10-23T00:00:00Z", "2023-10-23T23:59:59Z")
+| STATS median_duration = MEDIAN(event_duration) BY bucket
+// end::gs-bucket-stats-by-median[]
+| SORT bucket
+;
+
+median_duration:double | bucket:date
+3107561.0              |2023-10-23T12:00:00.000Z
+1756467.0              |2023-10-23T13:00:00.000Z
+;
+
+bucketByTimeDuration#[skip:-8.13.99, reason:BUCKET extended in 8.14]
+FROM sample_data
+| EVAL bucket = BUCKET(@timestamp, 30 minutes)
+| KEEP @timestamp, bucket
+| SORT @timestamp
+| LIMIT 4
+;
+
+       @timestamp:date  |         bucket:date
+2023-10-23T12:15:03.360Z|2023-10-23T12:00:00.000Z
+2023-10-23T12:27:28.948Z|2023-10-23T12:00:00.000Z
+2023-10-23T13:33:34.937Z|2023-10-23T13:30:00.000Z
+2023-10-23T13:51:54.732Z|2023-10-23T13:30:00.000Z
+;
+
+aggByTimeDuratinBucket#[skip:-8.13.99, reason:BUCKET extended in 8.14]
```

Comment:
super-duper-nit: Typo, although it was like that before already.
```suggestion
aggByTimeDurationBucket#[skip:-8.13.99, reason:BUCKET extended in 8.14]
```
Same for the `BIN` version.